KsApi/models/graphql/adapters/Project+FetchProjectQueryData.swift

```diff
@@ -48,14 +64,15 @@ extension Project {
         rewards: [noRewardReward(from: fragment)],
         addOns: nil,
         backing: nil,
-        currentUserChosenCurrency: data.me?.chosenCurrency
+        currentUserChosenCurrency: data.me?.chosenCurrency ?? configCurrency
```

Comment:
I imagine this is one of the more important bits to the PR?

Reply:
Yep, so this is something learned from how Android defaults their currency conversion. Basically before this we were using "USD" if the `data.me?.chosenCurency` was nil, which threw off all the conversions, but now because we're using an endpoint `v1/config` that gives us the users' default "global state" values, we can use its' currency instead.

Why did we miss this in the first place? Had no idea `v1/config` until Isa told me about it :)